egs2/ami/vad1/conf/tuning/decode_rnn.yaml

```diff
@@ -0,0 +1,2 @@
+maxlenratio: 0.0
+minlenratio: 0.0
```

Comment:
Do we need to control maxlen and minlen for the vad case? Since it is not an auto-regressive generation protocal, might be not?